bundles/org.openhab.binding.enocean/src/main/resources/OH-INF/i18n/enocean.properties

```diff
@@ -270,16 +270,17 @@ thing-type.config.enocean.rollershutter.enoceanId.description = EnOceanId of dev
 thing-type.config.enocean.rollershutter.pollingInterval.label = Polling Interval
 thing-type.config.enocean.rollershutter.pollingInterval.description = Time in seconds after a refresh is triggered, 0 to disable
 thing-type.config.enocean.rollershutter.receivingEEPId.label = EEP for Receiving States
-thing-type.config.enocean.rollershutter.receivingEEPId.option.A5_3F_7F_EltakoFSB = Eltako FSB14/61/71
+thing-type.config.enocean.rollershutter.receivingEEPId.option.A5_3F_7F_EltakoFSB = Eltako FSB14/61/71, FJ62 (4 byte 4BS status)
```

Comment:
Can you help me understand what is `(4 byte 4BS status)`? Is this for devices made before week 17/21?

Reply:
This means that this is a confirmation (status) telegram that is send as
ORG=0x07 -->  4 byte 4BS confirmation telegram
with 4 byte data/payload

see
Technical data of the wireless actuators, teach-in list, operating distances and contents of Eltako Wireless telegrams
[https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf](https://www.eltako.com/fileadmin/downloads/en/_main_catalogue/Gesamt-Katalog_ChT_gb_highRes.pdf)
Page T-16
CONFIRMATION TELEGRAMS OF BIDIRECTIONAL ACTUATORS.

FSB61NP-230V, FSB71, FJ62/12-36V DC, FJ62NP-230V

```
...
ORG =        0x07
Data_byte3 = driving time in 100 ms MSB
Data_byte2 = driving time in 100 ms LSB
Data_byte1 = 0x01 = driven up or 0x02 = driven down
Data_byte0 = 0x0A (not blocked) 
             or 
             0x0E (blocked)
...
```